ksqldb-rest-app/src/test/java/io/confluent/ksql/api/TlsTest.java

```diff
@@ -117,27 +122,47 @@ public void shouldReloadCert() throws Exception {
           TimeUnit.SECONDS.toMillis(1),
           TimeUnit.SECONDS.toMillis(1)
       );
+    } catch (final Throwable e) {
+      failed = true;
+      throw e;
     } finally {
       // restore cert regardless of failure above so as to not affect other tests
       // When: load valid store
-      SERVER_KEY_STORE.loadValidServerKeyStore();
-
-      assertThatEventually(
-          "Should successfully execute query with valid key store",
-          () -> {
-            // re-create client since server port changes on restart
-            this.client = createClient();
-
-            try {
-              return sendRequest("/query-stream", requestBody.toBuffer()).statusCode();
-            } catch (Exception e) {
-              return 0;
-            }
-          },
-          is(200),
-          TimeUnit.SECONDS.toMillis(1),
-          TimeUnit.SECONDS.toMillis(1)
-      );
+      SERVER_KEY_STORE.writeValidServerKeyStore();
+
+      if (!failed) {
```

Comment:
I've seen it take a number of seconds for the server to restart with the new cert after it's loaded. Even if the test fails, we should wait for the new cert to be loaded, which is why there's an `assertThatEventually` here, otherwise the next test that runs will likely fail.

Reply:
Actually, as I've changed `ServerKeyStore` to be non-static, there's no need for this to happen any more.  There's only one test in the class, and this class won't interfere with other classes... so no need to change, right?

Reply:
Looks like my comment from the previous PR wasn't addressed before merging: https://github.com/confluentinc/ksql/pull/6053#discussion_r474003954

I contend there _should_ be multiple tests running in this class 😄

Reply:
I'll sort that and this then ;)